proto/unit.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+import "common.proto";
+
+package dcs;
+
+message GetRadarRequest {
+  string name = 1;
+}
+
+message GetRadarResponse {
+  bool active = 1;
+  oneof target {
+    Unit unit = 2;
+    Weapon weapon = 3;
+    StaticObject static_object = 4;
+    SceneryObject scenery_object = 5;
+    Airbase airbase = 6;
+  }

```

Comment:
To we need a `null` counter-part? I guess that the target is `nil` at least in the case of `active` being `false`?

Reply:
It seems the recommendation and easiest approach is to just not include the field when the value is null so I am doing that.